src/vmm/src/version_map.rs

```diff
@@ -17,6 +18,7 @@ lazy_static! {
     pub static ref VERSION_MAP: VersionMap = {
         let mut version_map = VersionMap::new();
         version_map.new_version().set_type_version(DeviceStates::type_id(), 2);
+        version_map.new_version().set_type_version(BlockState::type_id(), 3);
```

Comment:
I think this commit should be squashed together with the one changing block state struct.

Reply:
Done.

Reply:
BlockState has version 2.

Reply:
Changed it.